src/com/ms/silverking/cloud/dht/daemon/MessageModule.java

```diff
@@ -636,7 +637,7 @@ public IPAndPort localIPAndPort() {
     @Override
     public boolean isLocal(IPAndPort replica) {
         if (debug) {
-            System.out.println("\t\t#### "+ replica +" "+ myIPAndPort +"\t"+ replica.equals(myIPAndPort));
+            Log.fineAsyncf("#### %s %s\t%s",replica, myIPAndPort, replica.equals(myIPAndPort));
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect